src/Visitor/CodeCoverageClassIgnoreVisitor.php

```diff
@@ -48,19 +48,17 @@ final class CodeCoverageClassIgnoreVisitor extends NodeVisitorAbstract
     public function enterNode(Node $node)
     {
         if (!$node instanceof Stmt\ClassLike) {
-            return null;
```

Comment:
it should be `null|int|Node` from interface's PHPDoc. Looks like incorrect change. Do we need to disable this rule?

Reply:
This was caused by `simplified_null_return` but I don't like the result either so reverted